src/Og.php

```diff
@@ -20,6 +20,21 @@
 class Og {
 
   /**
+   * The role name of the group non-member.
+   */
+ const OG_ANONYMOUS_ROLE = 'non-member';
```

Comment:
Shouldn't this be Og::ANONYMOUS_ROLE without the additional OG prefix? Same goes for the others.